From 3b36f9974e0d8b75c617b8938c41912e6b8779b0 Mon Sep 17 00:00:00 2001
From: P. Sadik <psadik@broadcom.com>
Date: Thu, 5 Dec 2013 16:58:46 +0530
Subject: tick: Enable skew_tick by default.

The networking performance improves with this change.
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/kernel/time/tick-sched.c b/kernel/time/tick-sched.c
index 35458ce..79d2594 100644
--- a/kernel/time/tick-sched.c
+++ b/kernel/time/tick-sched.c
@@ -1082,7 +1082,11 @@ static enum hrtimer_restart tick_sched_timer(struct hrtimer *timer)
 	return HRTIMER_RESTART;
 }
 
+#ifdef CONFIG_CPU_XLP
+static int sched_skew_tick = 1;
+#else
 static int sched_skew_tick;
+#endif
 
 static int __init skew_tick(char *str)
 {
-- 
1.7.1

